packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -179,8 +188,16 @@ export const DetailsTile = ({
   saveCount,
   headerText,
   title,
-  user
+  user,
+  track
 }: DetailsTileProps) => {
+  const isPremiumContentEnabled = useIsPremiumContentEnabled()
+  const { doesUserHaveAccess } = usePremiumContentAccess(
+    track ? (track as unknown as Track) : null
```

Comment:
agreed w reed re this cast – the `as unknown as x` cast is a bit of a smell, any way we can do this without this cast?

Reply:
agreed with yall, but i dont think it's a big deal here. We know the track is Track or SearchTrack, so if it exists we can just use the cast here. would spend more time refactoring this but i dont think it's hi-pri at this point...